WooCommerce/WooCommerceUITests/Mocks/__files/links.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>TESTING LINKS</title>
+</head>
+<body>
+    <h1>TESTING LINKS</h1>
+    <h2>Universal Links</h2>
+    <!-- Add universal links into this list for testing -->
+    <p><a href="https://woocommerce.com/mobile/payments">Payments</a></p>
```

Comment:
The person dealing with the test code in Xcode might not be very close to mocks and where they are. From this code:

```swift
ExternalAppScreen().openUniversalLinkFromSafariApp(link: "Payments")
```

it's not clear what is `Payments` link and where it is. WDYT of making the link text the same as href:

```suggestion
    <p><a href="https://woocommerce.com/mobile/payments">https://woocommerce.com/mobile/payments</a></p>
```

Reply:
good point! updated in https://github.com/woocommerce/woocommerce-ios/pull/9399/commits/d301d0dea9c5f48b060d91259040db9376a70a99